mucommander-core/src/main/java/com/mucommander/ui/viewer/FileEditorPresenter.java

```diff
@@ -152,24 +154,32 @@ public void windowClosing(WindowEvent e) {
                     KeyStroke.getKeyStroke(KeyEvent.VK_M, ActionEvent.CTRL_MASK),
                     (e) -> {
                         boolean fullScreen = getFrame().isFullScreen();
-                        getFrame().setFullScreen(!fullScreen);
-                        fullScreenMenuItem.setSelected(!fullScreen);
-                    }
-            );
-            fullScreenMenuItem.setSelected(getFrame().isFullScreen());
+                        switchFullScreenMode(!fullScreen);
+                    });
+
+            String windowWidthValue = EditorPreferences.WINDOW_WIDTH.getValue();
+            if (!"".equals(windowWidthValue)) {
```

Comment:
how about
```suggestion
            if (!windowWidthValue.isEmpty()) {
```
we don't allow null for EditorPreferences.WINDOW_WIDTH, right?